tests/container_checker/test_container_checker.py

```diff
@@ -238,3 +238,40 @@ def test_container_checker(duthosts, enum_rand_one_per_hwsku_hostname, enum_rand
         # Wait for 70s to 80s  such that Monit has a chance to write alerting message into syslog.
         logger.info("Sleep '{}'s to wait for the alerting message...".format(sleep_time))
         time.sleep(sleep_time)
+
+
+def test_container_checker_telemetry(duthosts, rand_one_dut_hostname):
```

Comment:
>test_container_checker_telemetry [](http://example.com/codeflow?start=4&length=32)

Since the test gap is identified on a slim image, will you specially test the HwSKU using slim image? Or, do you mean your general testcase will cover these HwSKU?

Reply:
For github sonic-buildimage release, we have used gnmi container to replace telemetry container.
So we need this test for all HwSKU.